onnx/defs/nn/defs.cc

```diff
@@ -197,6 +197,18 @@ ONNX_OPERATOR_SCHEMA(AveragePool)
         "average",
         "The output of each pooling window is divided by the number of elements exclude pad."));
 
+ONNX_OPERATOR_SCHEMA(AveragePool)
+    .SinceVersion(7)
+    .FillUsing(PoolOpSchemaGenerator(
+        "AveragePool",
+        "average",
+        "The output of each pooling window is divided by the number of elements (exclude pad when attribute count_include_pad is zero)."))
+    .Attr(
+        "count_include_pad",
```

Comment:
Should we add something similar to MaxPool? So if all elements are negatives out zeros.

Reply:
Yeah for the sake of leaving no ambiguity, let's add that as well (in a separate PR).